Logos/cisco-logo-72px.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="72px" height="72px" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Remove xmlns:xlink="http://www.w3.org/1999/xlink" from .svg file. 

Guidance is available at:

https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/ReadMe.md

Ensure raw file of logo does not have any of the following:
cls and style formats
embedded png formats
xmlns:xlink
data-name

Reply:
complete